docker-compose.integration-tests.yml

```diff
@@ -23,9 +23,8 @@ services:
       - "0.0.0.0:6432:5432"
 
   mysql-test:
-    image: mysql:8
```

Comment:
i see here that this one _had_ been pinned. again, i think it'd just be better to remove.

as we discussed offline, i'm just generally not sure what this compose YML is being used for anyway, so we could perhaps remove this entry all together - but we also don't _need_ to in this increment, if we want to play it safe and keep it as as simple a change as possible :)

Reply:
Let me try removing it and see if it causes problems, I definitely pinned my `9` after seeing that `8` but a pinned and un-run `8` isn't useful

Reply:
removed here: https://github.com/ethyca/fides/pull/5070/commits/5b10b02bbd03b8d02610c7aaf7b4198c3b0847c1